website/docs/r/ovh_domain_zone_record.html.markdown

```diff
@@ -46,9 +46,9 @@ The following attributes are exported:
 * `ttl` - The TTL of the record
 
 ## Import
+In order to import in terraform an existing zone record, you will need its _{id}_ which can be retrieved by using [OVH API portal](https://api.ovh.com/console/#/domain/zone/%7BzoneName%7D/record~GET)
```

Comment:
```suggestion
OVHcloud domain zone record can be imported using its `ids`, which can be retrieved by using `domain_zone` datasource, and the `zone`, separated by "." E.g.,
```

Reply:
We try to keep consistency with other documentation pages and use existing resources or datasources when it's possible :)